wiki/release/release-process.md

```diff
@@ -412,35 +421,19 @@ $ twine upload pulsar_client-*.whl
 
 ### MacOS
 
-> **NOTES**
->
-> You need to install following softwares before proceeding:
->
-> - [VirtualBox](https://www.virtualbox.org/)
-> - [VirtualBox Extension Pack]
-> - [Vagrant](https://www.vagrantup.com/)
-> - [Vagrant-scp](https://github.com/invernizzi/vagrant-scp)
->
-> And make sure your laptop have enough disk spaces (> 30GB) since the build scripts
-> will download MacOS images, launch them in VirtualBox and build the python
-> scripts.
-
-Build the python scripts.
+There is a script that builds and packages the Python client inside Docker images.
 
 ```shell
-$ cd pulsar-client-cpp/python/pkg/osx/
-$ ./generate-all-wheel.sh v2.X.Y
+$ pulsar-client-cpp/python/build-mac-wheels.sh
 ```
 
 The wheel files will be generated at each platform directory under `pulsar-client-cpp/python/pkg/osx/`.
 Then you can run `twin upload` to upload those wheel files.
 
-If you don't have enough spaces, you can build the python wheel file platform by platform and remove the images under `~/.vagrant.d/boxes` between each run.
-
-## 13. Publish MacOS libpulsar package
+## 12. Publish MacOS libpulsar package
```

Comment:
This is similar with Helm Chart, it can be skipped if the major version number is not latest.

Reply:
Why? libpulsar in Homebrew could have multiple versions at the same time, but there is only one latest version in the charts repo.

Reply:
1. I don't see how to install old version of `libpulsar` with `brew`
2. There is no record of non-latest version libpulsr PR in 
[homebrew-core repo history](https://github.com/Homebrew/homebrew-core/pulls?q=is%3Apr+libpulsar+is%3Aclosed)

Reply:
@chenrui333 Can you help verify this? since you released past few versions of `libpulsar`.

Reply:
Oh I see. I thought it's similar with `pip`.